src/containers/sidebars/aoi-sidebar/component.jsx

```diff
@@ -41,6 +41,7 @@ import styles from './styles.module.scss';
 
 import { ReactComponent as ClimateRegimeIcon } from 'icons/climate-regime.svg';
 import { ReactComponent as EditIcon } from 'icons/edit.svg';
+import { ReactComponent as GoalGlobeIcon } from 'icons/goal-globe.svg';
```

Comment:
This could probably be smaller on a png. Can you check? Svg are reserved for icons mainly